### ISSUE TITLE
Range.__iter__ iterates wrong cells

### ISSUE THREAD

**Issue by albertix** [ZoomerAnalytics/xlwings#595]
with code

    for cell in xlwings.Range('A1:A3, C2:C3, D4'):
        print(cell.address)

i got

    $A$1
    $A$2
    $A$3
    $A$4
    $A$5
    $A$6
------------
which need to be

    for cell in xw.Range('A1:A3, C2:C3, D4').impl.xl.cells:
        print(cell.address)

    $A$1
    $A$2
    $A$3
    $C$2
    $C$3
    $D$4
------------
Sorry about my not so excellent English.

**Comment by fzumstein** [comment]
That's correct, xlwings doesn't support [multiple ranges](https://msdn.microsoft.com/en-us/library/office/ff194061.aspx) (yet?).

Note that I fixed your 2nd sample to use `api` instead of `impl.xl`, the official way of getting the underlying pywin32 object.

**Comment by akasolace** [comment]
for my knowldege,
why is commit made by Albertix not been merged ? The solution cannot be considered closed ?

**Comment by fzumstein** [comment]
he never made a pull request...

**Comment by akasolace** [comment]
ok I just did a PR

**Comment by fzumstein** [comment]
I don't think so...nothing showing up: https://github.com/ZoomerAnalytics/xlwings/pulls

**Comment by akasolace** [comment]
it should be ok now
